CHANGELOG

```diff
@@ -1,3 +1,4 @@
+[#90] OpenFaas is not serializing/deserializing query answers
```

Comment:
New entries in CHANGELOG ared supposed to be added at the end of the file, not at the begining